packages/components/src/components/hds/modal/footer.ts

```diff
@@ -6,9 +6,19 @@
 import Component from '@glimmer/component';
 import { deprecate } from '@ember/debug';
 
-export default class HdsModalFooterComponent extends Component {
-  constructor() {
-    super(...arguments);
+export interface HdsModalFooterSignature {
+  Args: {
+    onDismiss: (event: MouseEvent) => void;
```

Comment:
question: @alex-ju should the onDismiss/close args be required? thinking about it more, I'm guessing they're optional.

Reply:
we can mark it as optional, indeed. it's not part of the public API anyway, more of a way for us to pass the close function to the footer subcomponent